Makefile.config

```diff
@@ -201,17 +201,16 @@ ifneq ($(CXX),icc)
     ifdef CMSSW_BASE
       CPPFLAGS += -I$(shell cd $$CMSSW_BASE && scram tool tag tbb INCLUDE)
       LDFLAGS_HOST += -L$(shell cd $$CMSSW_BASE && scram tool tag tbb LIBDIR)
-    else ifneq (,$(realpath /opt/rh/python27/root/usr/include))
-      CPPFLAGS += -I/opt/rh/python27/root/usr/include
-      LDFLAGS_HOST += -L/opt/rh/python27/root/usr/lib64
+    else ifdef TBB_GCC
+      TBB_PREFIX = $(TBB_GCC)
     endif
   endif
 endif
 
 ifdef WITH_TBB
   # icc finds tbb in its own installation, but allow overriding in case it doesn't
   ifdef TBB_PREFIX
-    CPPFLAGS += -I${TBB_PREFIX}/include
+    CPPFLAGS += -I${TBB_PREFIX}/include/tbb
```

Comment:
Why is `/tbb` needed here? Our `#include`s are already like `#include <tbb/tbb.h>`, no?

Reply:
Doesn't seem to make a difference either way, but you're right, I'll fix this